contrib/cube/meson.build

```diff
@@ -11,13 +11,15 @@ cube_scan = custom_target('cubescan',
 )
 generated_sources += cube_scan
 cube_sources += cube_scan
+bc_cube_sources = cube_sources
 
 cube_parse = custom_target('cubeparse',
   input: 'cubeparse.y',
   kwargs: bison_kw,
 )
 generated_sources += cube_parse.to_list()
 cube_sources += cube_parse
+bc_cube_sources += cube_parse[0]
```

Comment:
This looks a bit suspect to me. Are you sure this is correct? I would have expected the lines to not have three different variants?

Reply:
The output of bison is two files, a .c and a .h, I can't use .h as input for the bitcode generation so that's why I include only the first output, you can check `bison_kw` for more info.